packages/cozy-harvest-lib/src/helpers/konnectors.js

```diff
@@ -78,6 +79,16 @@ export class KonnectorJobError extends Error {
   }
 }
 
+/**
+ * Returns the account type. Based on the information container in the oauth prop,
```

Comment:
```suggestion
 * Returns the account type. Based on the information from the oauth attribute,
```

I think we should keep "prop" for when we talk about React components, I was confused.

Reply:
Actually, we're talking about the manifest property here. I will clarify.

Reply:
My point was that I prefer to use attribute instead of property.

Reply:
Ah yes sorry. I totally agree.